db/migrate/20220725145007_create_account_credit_cards.rb

```diff
@@ -0,0 +1,16 @@
+class CreateAccountCreditCards < ActiveRecord::Migration[6.0]
+  def change
+    create_table :account_credit_cards do |t|
+      t.string :bank, null: false
+      t.string :brand, null: false
+      t.integer :last_digits
+      t.float :credit_limit, precision: 10, scale: 2, null: false, default: 0
+      t.string :owner_name
+      t.integer :due_day
+      t.date :expires_at
```

Comment:
Maybe we can put these fiels with `null: false` too:

```suggestion
      t.integer :due_day
      t.date :expires_at
```